api_tests/tests/siren_schema.ts

```diff
@@ -24,7 +30,7 @@ async function assertValidSirenDocument(
     const swapResponse = await alice.cnd.fetch(selfLink);
     const swapEntity = swapResponse.data as siren.Entity;
 
-    expect(swapEntity).toMatchSchema(sirenJsonSchema);
+    expect(swapEntity).toMatchSchema(sirenSwapJsonSchema);
     expect(swapEntity.properties).toMatchSchema(swapPropertiesJsonSchema);
```

Comment:
This is the one that you can actually delete given that we no longer provide these properties.